composer.json

```diff
@@ -29,7 +29,7 @@
       "homepage": "https://mvorisek.cz/"
     }
   ],
-  "version": "dev-develop",
+  "version": "fix-version",
```

Comment:
this is of course invalid to be commited into develop

Reply:
sorry, fixed